corehq/apps/accounting/forms.py

```diff
@@ -137,6 +141,7 @@ def __init__(self, account, *args, **kwargs):
                 'email_list': ','.join(contact_info.email_list),
                 'is_active': account.is_active,
                 'is_customer_billing_account': account.is_customer_billing_account,
+                'billing_admin_emails': ', '.join(account.billing_admin_emails),
```

Comment:
The space after the comma is unnecessary - see ```email_list```.  Not sure if it would introduce problems.